apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/S3MinioConfig.java

```diff
@@ -0,0 +1,31 @@
+package gov.cms.bfd.pipeline.ccw.rif.extract.s3;
+
+public class S3MinioConfig {
+
+  public String minioUserName;
+  public String minioPassword;
+  public String minioEndpointAddress;
+  public Boolean useMinio;
+
+  private static S3MinioConfig single_instance = null;
+
+  // Constructor of this class
+  // Here private constructor is is used to
+  // restricted to this class itself
+  private S3MinioConfig() {
+    minioUserName = System.getProperty("s3.localUser", "bfdLocalS3Dev");
+    minioPassword = System.getProperty("s3.localPass", "bfdLocalS3Dev");
+    minioEndpointAddress = System.getProperty("s3.localAddress", "http://localhost:9000");
+    useMinio = Boolean.parseBoolean(System.getProperty("s3.local", "false"));
```

Comment:
This can be accomplished with the slightly more compact `Boolean.getBoolean("s3.local")`